src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Configuration/TargetFrameworkProjectConfigurationDimensionProviderTests.cs

```diff
@@ -71,7 +75,8 @@ public class TargetFrameworkProjectConfigurationDimensionProviderTests
             using (var projectFile = new MsBuildProjectFile(projectXmlTFMs))
             {
                 IProjectXmlAccessor _projectXmlAccessor = IProjectXmlAccessorFactory.Create(projectFile.Project);
-                var provider = new TargetFrameworkProjectConfigurationDimensionProvider(_projectXmlAccessor);
+                ITelemetryService _telemetryService = ITelemetryServiceFactory.Create();
```

Comment:
shouldn't there be at least one test that make sure telemetry is actually called? so that we don't accidentally remove telemetry?